api/search.go

```diff
@@ -266,6 +312,7 @@ func parseOffset(ctx context.Context, params url.Values, validator QueryParamVal
 }
 
 func parseAndValidateContentTypes(ctx context.Context, params url.Values) (contentTypes []string, err error) {
+	// read content type (expected CSV value), with default, to make sure some content types are
```

Comment:
to make sure some content types are..?

Reply:
Old message that got brought across, not sure when it half it got removed, removed it all now anyways!